docs/source/tutorial_quick.md

```diff
@@ -91,14 +91,14 @@ Let's begin again by printing the first few examples.
 
 ```bash
 # display first examples from twitter dataset
-parlai display_data -t twitter
+parlai display_data --task twitter
 ```
 
 Now, we'll train the model. This will take a while to reach convergence.
 
 ```bash
 # train transformer ranker
-parlai train_model -t twitter -mf /tmp/tr_twitter -m transformer/ranker -bs 16 -vtim 3600 -cands batch -ecands batch --data-parallel True
+parlai train_model --task twitter --model-file /tmp/tr_twitter --model transformer/ranker --batchsize 16 --validation-every-n-secs 3600 --candidates batch --eval-candidates batch --data-parallel True
```

Comment:
`validation-every-n-secs` is too long. i know this is the readme, but is that possible to change that into something like `validation-freq`

Reply:
that's a valid point (no pun intended) but the issue is that we have several (and soon to be more) ways of defining "frequency" (epochs, seconds, soon-to-be training steps).

perhaps we can change them all to `--validation-freq-secs`, `--validation-freq-steps`, `--validation-freq-epochs`

Reply:
I'm hesitant to change the names of options that have been around 3 years :/